src/python/pants/backend/python/util_rules/pex.py

```diff
@@ -453,6 +462,49 @@ class _BuildPexRequirementsSetup:
     concurrency_available: int
 
 
+@dataclass(frozen=True)
+class ReqStrings:
+    req_strings: tuple[str, ...]
+
+
+@rule
+async def get_req_strings(pex_reqs: PexRequirements) -> ReqStrings:
+    addrs: list[Address] = []
+    specs: list[str] = []
+    req_strings: list[str] = []
+    for req_str_or_addr in pex_reqs.req_strings_or_addrs:
+        if isinstance(req_str_or_addr, Address):
+            addrs.append(req_str_or_addr)
+        else:
+            assert isinstance(req_str_or_addr, str)
+            # Require a `//` prefix, to distinguish address specs from
+            # local or VCS requirements.
+            if req_str_or_addr.startswith(os.path.sep):
```

Comment:
This should probably be double `//`, otherwise it's ambiguous with what someone might try as an absolute file path.

Reply:
An absolute file path is definitely not any kind of valid requirement, but sure.